### PR TITLE
Is Send test

### PR DESCRIPTION
**Pull request by audioXD**
Added a test for all public async methods to check if they are Send. It works on the principle it compiles it passes.

### CODE REVIEW

**Comment by audioXD** [comment]
About #4

**Review by stevepryde — COMMENTED** [review]
This should probably go in a #[cfg(test)] block as an actual test

**Comment by audioXD** [comment]
I didn't do that because I don't want to run the code even while testing.

File: src/query.rs [hunk 1]
```diff
@@ -691,3 +691,32 @@ impl ElementQueryable for WebDriver {
         ElementQuery::new(ElementQuerySource::Driver(&self.session), poller, by)
     }
 }
+
+//
+// Send test (It passes if it compiles)
+//
+
+async fn _test_is_send() -> WebDriverResult<()> {
```
Comment (stevepryde) [inline src/query.rs]: Consider removing the blank line to make it more obvious that the comment belongs with the function. Should it have /// instead?

Also please add an extra sentence to make it clearer that this code is not intended to be executed. Thanks

**Comment by audioXD** [comment]
Oh... you ment just adding #[cfg(test)]  without the:
```
mod test {
    #[test]
    fn test() {}
}
```

Sorry I misunderstoo. :p

**Comment by stevepryde** [comment]
> Oh... you ment just adding #[cfg(test)] without the:
> 
> ```
> mod test {
>     #[test]
>     fn test() {}
> }
> ```

Well, I did originally mean to put it inside `mod test` etc - but I agree with your reason for not doing that. However, maybe it should still conditionally (not) compile

**Comment by audioXD** [comment]
Yes I agree

**Review by stevepryde — APPROVED** [review]
Looks great

**Pull request merged**